README.md

```diff
@@ -171,6 +176,8 @@ This tool currently is only useful for packing assets.
         actually executed parallel yet)
       * Handles multiple views and phases allowing advanced features like shadow mapping
       * Flexible sorting mechanism for interleaving and batching write commands from multiple rendering features
+   * Visibility Region - Built on top of `rafx-visibility`
+      * Provides reference counted wrappers around the low-level handles and a more ergonomic interface that assumes a clear definition of `static` and `dynamic` game geometry.
```

Comment:
I didn't really understand what "assumes a clear definition of `static" means. I think this might also be too much detail. "Higher-level visibility API, integrated with framework render jobs" or something like that I think is enough.